src/utils/itrm/land-valuation.util.ts

```diff
@@ -1,36 +1,32 @@
 ﻿import {Result} from "../../types/common.type";
-import {Raise} from "../common.util";
+import {Raise, RemoveUndefinedProperties} from "../common.util";
 import axios from "axios";
 import {LogError} from "../logging.util";
 import {Module} from "../../enums/logging.enum";
 import {
-  LandValuationDatum,
   MetaverseGlobalData,
   MetaversePerformance
 } from "../../interfaces/itrm/land-valuation.interface";
 import {Metaverses} from "../../enums/metaverses.enum";
-
-// type ApiMetaverse = 'sandbox' | 'decentraland' | 'fluf' | 'somnium-space';
+import { SingleLandAPIResponse } from "../../types/valuationTypes";
 
 interface MapParams {
   from: number;
   size: number;
-  tokenId?: number;
+  tokenId?: string;
   x?: number;
   y?: number;
   reduced?: boolean;
 }
 
-export async function GetMapLandValuation(metaverse: Metaverses, params: MapParams): Promise<Result<Record<string, LandValuationDatum>>> {
+export async function GetMapLandValuation(metaverse: Metaverses, params: MapParams): Promise<Result<Record<string, SingleLandAPIResponse>>> {
```

Comment:
You just returned back to old type (and by old i mean old)
fix plz

Reply:
landValuationDatum deleted